bb/parser.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Bloomsbury
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  // let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+  let match;
+
+  if ((match = /^\/book\/([a-z-]+)\.pdf$/i.exec(path)) !== null) {
+    // //www.bloomsburycollections.com/book/metaphors-of-death-and-resurrection-in-the-quran-an-intertextual-approach-with-biblical-and-rabbinic-literature.pdf?dl
```

Comment:
you should add [0-9]+ to the parser. Browsing quickly on the platform, I saw that there were titles with numbers

example : 
- https://www.bloomsburycollections.com/book/cesare-zavattini-selected-writings-volume-1/
- https://www.bloomsburycollections.com/book/cesare-zavattini-selected-writings-volume-1.pdf